README.md

```diff
@@ -13,7 +13,7 @@ Netlify hosted:
 [//]: # (numfocus-fiscal-sponsor-attribution)
 
 The Insight Software Consortium (ISC) uses an [open governance model](./GOVERNANCE.md)
-and is fiscally sponsored by [NumFOCUS](https://numfocus.org/). Consider making a [tax-deductible donation](https://numfocus.org/donate-to-itk)
+and is fiscally sponsored by [NumFOCUS](https://numfocus.org/). Consider making a [tax-deductible donation]()https://opencollective.com/itk
```

Comment:
Syntax error here for the closing bracket.

I guess this change should also be made to ITK, ITKExamples, ITK SW Guide, SimpleITK.

Reply:
:+1: 

I will also make PR's to the other repositories.

Reply:
I made a PR against ITK. https://github.com/InsightSoftwareConsortium/ITK

I did not find links on ITKSphinxExamples, SimpleITK, ITKSoftwareGuide.

Reply:
OK, maybe we discussed this at some point and preferred not to add it. I cannot remember. Thanks Matt.

Resolving.